src/main.ts

```diff
@@ -10,6 +10,12 @@ import "primevue/resources/primevue.min.css";
 import "primeicons/primeicons.css";
 import "leaflet/dist/leaflet.css";
 
+// type D = L.Icon.Default & {
+//     _getIconUrl?: string;
+//   };
+  
+// delete (L.Icon.Default.prototype as D)._getIconUrl;
```

Comment:
potrzebne?

Reply:
szczerze powiedziawszy pomyślałem że znalazło się to tam przy okazji jednego z dzielonych sesji w vscodzie i zostawiłem jeżeli okazało się to jakimś niedokończonym wątkiem. Na razie może zostać i najwyżej usunę przy okazji